electrum_nmc/electrum/names.py

```diff
@@ -193,9 +193,12 @@ def calc_truncated_checksum(public_key):
 
 def validate_zeronet_address(address: str) -> None: # Validate P2PKH Address (Zero Net)
     try:
-        base58.b58decode_check(address)
+        addrtype, _ = b58_address_to_hash160(address)
```

Comment:
Preferred using the `b58_address_to_hash160` function, since the other return True/False while this one raises exceptions.